src/pages/CreateAccount.jsx

```diff
@@ -0,0 +1,193 @@
+import { useState, useRef } from 'react';
+import { Form, Button, Card, Alert } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import FilterDropdown from '../components/common/FilterDropdown';
+
+function CreateAccount() {
+  // @TODO Fetch the schools from the database
+  // For now mock the data
+  const schools = [
+    'Tufts University',
+    'Ohio State University',
+    'Tower Hill School',
+    'Wesleyan University'
+  ];
+  const [school, setSchool] = useState(schools[0]);
+  const schoolEmailRef = useRef();
+  const schoolEmailConfirmRef = useRef();
+  const passwordRef = useRef();
+  const passwordConfirmRef = useRef();
+  // const { signup } = useAuth();
+  const [preferredContact, setPreferredContact] = useState({
+    schoolEmail: false,
+    secondaryEmail: false,
+    phone: false
+  });
+  const secondaryEmailRef = useRef();
+  const phoneRef = useRef();
+  const [paymentMethod, setPaymentMethod] = useState({
+    cash: false,
+    venmo: false
+  });
+  const venmoRef = useRef();
+  const [error, setError] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  async function handleSubmit(e) {
+    // prevent the form from refreshing
+    e.preventDefault();
+    // do the validation checks
+
+    if (passwordRef.current.value !== passwordConfirmRef.current.value) {
+      setError('Passwords do not match');
+    }
+
+    if (schoolEmailRef.current.value !== schoolEmailConfirmRef.current.value) {
+      setError('Emails do not match');
+    }
+
+    if (
+      !(preferredContact.schoolEmail || preferredContact.secondaryEmail || preferredContact.phone)
+    ) {
+      setError('No preferred contact method selected');
+    }
+
+    if (!(paymentMethod.cash || paymentMethod.venmo)) {
+      setError('No preferred payment method selected');
+    }
+
+    try {
+      // setError('');
+      // set up a load state, so when signing up the user, we disabled the "Sign Up" botton below,
+      // so they don't automatically keep clicking the button and create multiple of accounts at the same time
+      setLoading(true);
+      // await signup(schoolEmailRef.current.value, passwordRef.current.value);
+    } catch {
+      setError('Failed to create an account');
+    } finally {
+      setLoading(false);
+    }
+  }
+
+  const handlePreferredContact = (e) => {
+    const { checked, name } = e.target;
+    setPreferredContact({
+      ...preferredContact,
+      [name]: checked
+    });
+  };
+
+  const handlePreferredPayment = (e) => {
+    const { checked, name } = e.target;
+    setPaymentMethod({
+      ...paymentMethod,
+      [name]: checked
+    });
+  };
+
+  return (
+    <>
```

Comment:
Try use Bootstrap for styling (refer to the LoginPage.jsx). You can wrap your <Card> component with a container and use the grid system to center it.